tests-ng/plugins/booted.py

```diff
@@ -1,24 +1,29 @@
 import pytest
+from typing import Any, List
 
 system_booted = False
 
-def is_system_booted():
+
+def is_system_booted() -> bool:
     return system_booted
 
-def pytest_addoption(parser):
+
+def pytest_addoption(parser: pytest.Parser):
     parser.addoption(
         "--system-booted",
         action="store_true",
         help="Set if the system under test was booted instead of running in a chroot or container. This will enable kernel, systemd, and other system level tests."
     )
 
-def pytest_configure(config):
+
+def pytest_configure(config: pytest.Config):
     global system_booted
     system_booted = config.getoption("--system-booted")
 
     config.addinivalue_line("markers", "booted: mark test to run only on a booted target, i.e. not in a container or chroot")
 
-def pytest_collection_modifyitems(config, items):
+
+def pytest_collection_modifyitems(config: pytest.Config, items):
```

Comment:
`items: List[Any]` or `Any` if we don't know for sure that the item's value is a `pytest.Item` object.

Reply:
It should be properly defined _somewhere_ what type items is here, right? so `List[Any]` wouldn't make too much sense.